catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -50,11 +50,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.4</version>
-        </dependency>
-        <dependency>
-            <groupId>org.codice.ddf.registry</groupId>
-            <artifactId>registry-schema-bindings</artifactId>
+            <version>3.0</version>
```

Comment:
Seeing commons-lang3 used in a few places, we should consider doing dependency management on it. Why did you drop the version from 3.4 to 3.0?